src/chapters/aspdotnet-model-validation/validation-attributes.rst

```diff
@@ -0,0 +1,184 @@
+Validation Attributes
+=====================
+
+Within the model of a C# web application, we can define validation rules using attributes.
+Validation attributes can be applied to model fields. 
+
+.. index::
+   single: validation, annotations
+
+.. index:: ! [Required], ! [EmailAddress], ! [Range], ! [StringLength]
+
+Common Attributes
+------------------
+
+We'll use only a few of these attributes, but you can find a full list in the `documentation <https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-3.1#built-in-attributes>`_.
+
+.. list-table:: Common Validation Annotations
+   :header-rows: 1
+
+   * - Annotation
+     - Description
+     - Syntax
+   * - ``[Required]``
+     - Specifies that a field cannot be null
+     - ``[Required]``
+   * - ``[Range]``
+     - Specifies the range of potential values of a numeric field.
+     - ``[Range(0,100)]``
+   * - ``[StringLength]``
+     - Specifies the maximum length of a string field. Has additional optional parameters to specify the minimum length of a string field.
+     - ``[StringLength(100)]``
+   * - ``[EmailAddress]``
+     - Specifies that a string field should conform to email formatting standards.
+     - ``[EmailAddress]``
+
+
+.. admonition:: Example
+
+   To apply the validation rules of the :ref:`example on the previous page <validation-example>` to the fields of a ``User`` model class, we can use ``@Size`` and ``@NotBlank``.
```

Comment:
This sentence references Java validation annotations, rather then the C# attributes used in the code.

Reply:
Fixed!